corehq/apps/export/models/new.py

```diff
@@ -1390,7 +1390,7 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
         try:
             current_schema = cls._reorder_schema_from_app(current_schema, app_id, identifier)
         except Exception as e:
-            _soft_assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'))
+            _soft_assert = soft_assert(notify_admins=True)
```

Comment:
ok changed to a log message so it shows up in sentry. https://github.com/dimagi/commcare-hq/pull/17219/commits/d2f97a66c00005d3b055307c80ada1c67e7297a0

these don't happen often (i think i've only received a couple). i think it's good to log somewhere that apps are failing to be processed by exports